MAINTAINERS.md

```diff
@@ -54,8 +54,8 @@ If you are an owner of the organization, you should be able to see an automated
 
 ## Organization members
 
-In addition everything that a repository collaborator can do,
-an organization member also has write access to all the repositories in the tldr-pages organization.
+In addition to everything that repository collaborators can do, organization members
+have collaborator access to all the repositories in the tldr-pages organization.
```

Comment:
The phrasing changes are 👍, but while I agree that "write access" is not clear enough, I'm not sure organization members can be described to simply have collaborator access to all repos in the org. I believe they have [additional permissions](https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-organizations-and-teams/permission-levels-for-an-organization), though the exact diff is not exactly clear on my mind.

Perhaps it's not important to be exhaustive here. Maybe we can just add a link here, something like this?

```suggestion
have collaborator access to all the repositories in the tldr-pages organization,
in addition to [a few extra maintenance tools](https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-organizations-and-teams/permission-levels-for-an-organization).
```

Reply:
Okay.  Interesting.